docs/releases/2.1.txt

```diff
@@ -153,7 +153,8 @@ Migrations
 Models
 ~~~~~~
 
-* ...
+* Made ``ModelBase.__new__`` pass ``**kwargs`` to ``type.__new__``
```

Comment:
It would be more appropriate to describe the new functionality rather than the implementation details (similarly, for the Trac ticket summary and the commit message).

The ticket reference isn't needed here (that's only for bug fixes in minor release versions).